cookbook/deployment/deploying_workflows.py

```diff
@@ -2,6 +2,8 @@
 Deploying Workflows - Registration
 ----------------------------------
 
+.. tags:: Deployment, Advanced
```

Comment:
I feel like deploying workflows would be Basic or Intermediate? seems like a core part of what Flyte enables